src/test/java/com/github/iveina2021/api_test/UpdateUserAccountTest.java

```diff
@@ -0,0 +1,52 @@
+package com.github.iveina2021.api_test;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class UpdateUserAccountTest {
+
+    @BeforeEach
+    public void setUp() {
+        RestAssured.baseURI = "https://automationexercise.com";
+    }
+
+    @Test
+    public void updateUserAccount() {
+
+        Map<String, String> payload = new HashMap<>();
+
+        payload.put("name", "Leonard7");
```

Comment:
Let's avoid duplication in this request payload.

Options: 

1) we can extract method e.g. createUserRequest(...) and specify all parameters as argument - not so good solution because we will add 10+ arguments to single method

2) We can use Builder pattern. It's widely used in industry.

IDE can generate builder for class. Let's check how to do it offline.

Reply:
Added Builder